tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -130,35 +130,10 @@ var _ = Describe("odo devfile catalog command tests", func() {
 	})
 
 	Context("When executing catalog list components with experimental mode set to true", func() {
-
-		componentName := "nodejs"
-
 		It("should prove that nodejs is present in both S2I Component list and Devfile Component list", func() {
-
 			output := helper.CmdShouldPass("odo", "catalog", "list", "components", "-o", "json")
-
-			wantOutput := []string{componentName}
-
-			var data map[string]interface{}
-
-			err := json.Unmarshal([]byte(output), &data)
-
-			if err != nil {
-				Expect(err).Should(BeNil())
-			}
-			outputBytes, err := json.Marshal(data["s2iItems"])
-			if err == nil {
-				output = string(outputBytes)
-			}
-
-			helper.MatchAllInOutput(output, wantOutput)
-
-			outputBytes, err = json.Marshal(data["devfileItems"])
-			if err == nil {
-				output = string(outputBytes)
-			}
-
-			helper.MatchAllInOutput(output, wantOutput)
+			err := utils.VerifyCatalogListComponent(output)
```

Comment:
Pass the component name too as an argument to avoid hardcoding in the library function
```
err := utils.VerifyCatalogListComponent(output, componentName)
```